### PR TITLE
Fixed Hello World in Korean

### PR DESCRIPTION
**Pull request by dwkim922**
The Korean was wrong, seemed like it was done with a translator. So I… changed it to the correct term, since Ive learnt programming in Korean before Im certain that this is right.

5020d46aaf94dd966a2e88b96ba7c143e3c778425b4329e49bac3d9c46829253

### CODE REVIEW

File: src/containers/Home/HomeHero/hello-world.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ const HelloWorld: {[isoLangCode: string]: string} = {
   he: 'שלום עולם',
   hi: 'नमस्ते दुनिया',
   id: 'Halo Dunia',
-  ko: '여보세요 세계',
+  ko: '안녕 세계야',
```
Comment (angle943) [inline src/containers/Home/HomeHero/hello-world.ts:9]: @dwkim922 hey is it "안녕 세계야" or "안녕 세계" ?

Comment (dwkim922) [reply src/containers/Home/HomeHero/hello-world.ts:9]: Hey @angle943. 
Actually, "안녕 세상아" would sound better.

Comment (dwkim922) [reply src/containers/Home/HomeHero/hello-world.ts:9]: @angle943 should i pull request again with "안녕 세상아"?

Comment (angle943) [reply src/containers/Home/HomeHero/hello-world.ts:9]: it's cool, i'll update it on this PR once i get around it. thanks!

**Pull request merged**